testsuite/features/secondary/srv_cobbler_buildiso.feature

```diff
@@ -71,8 +70,12 @@ Feature: Cobbler buildiso
 
   Scenario: Cleanup: delete test distro and profiles in the cobbler buildiso context
     Given I am authorized as "testing" with password "testing"
+    # the order is important here. First systems, then profiles then distros
```

Comment:
Let's have a scenario for each of them?
- Systems
- Profiles
- Distros

Reply:
Sure. I can create a scenario for each of. This will make reading the test suite reports and debugging easier.

Reply:
Fixed.